src/js/components/Button/Button.js

```diff
@@ -1,8 +1,9 @@
-import React, { cloneElement, Children, Component } from 'react';
+import React, { cloneElement, Children, useState } from 'react';
 import { compose } from 'recompose';
 
 import { withTheme } from 'styled-components';
 
+import PropTypes from 'react-desc/lib/PropTypes';
```

Comment:
why do we suddenly need the PropTypes? we didn't need it before the hooks.

Reply:
@ShimiSun we don't necessarily need it I just put it out there since we were declaring the default PropTypes at the bottom of this file so added the prop types for those values.

Reply:
proptypes are handled by the doc file if i am not mistaken. i think we shouldn't add them here as well.

Reply:
Sounds good I got rid of the proptypes!